### ISSUE TITLE
MoU opt-out

### ISSUE THREAD

**Issue by mnot** [w3c/w3process#753]
It is very odd that the MoU process is effectively opt-out -- i.e., the AC has to initiate an appeal, whereas every other technical decision is opt-in. What motivates this?

**Comment by frivoal** [comment]
The details have been tweaked, but the fundamental opt-out nature of this isn't new (introduced [20 years ago](https://www.w3.org/2003/06/Process-20030618/liaisons.html#Liaisons) as far as I can tell). Since I wasn't there when this was introduced, I can only speculate as to why that is the way it is.

My guess is that the reason is that it is not, per se, a technical decision, but rather an operational one. The Team is choosing to liaise (or similar) with some other organization.

Also, it isn't the only thing that's opt-out by default). So are: Charter extensions, change of Team Contact or Chair, acknowledgement/rejection of member submissions, and an number of other Team or Chair decision where the explicit appeal path isn't listed, but which can be formally objected against, as any decision can. Group closure also used to be (until this Proposed 2023).

**Comment by dwsinger** [comment]
We didn't fiddle with the actual process here; the changes were to disambiguate what lies with overall governance and when the AC should be involved. Perhaps it's also due for a look at substance…

**Comment by frivoal** [comment]
We did fiddle with the process a little bit. In addition to [disambiguation between what's up for the AC and what's Board territory](https://github.com/w3c/w3process/pull/704), we also added [an explicit advance notice period for MoUs](https://github.com/w3c/w3process/pull/619). But neither of these change the fact that, when applicable to them, the AC gets an opt-out, not an opt-in. That's old.

**Comment by mnot** [comment]
I agree this needs another look, but would be satisfied if the resolution were to defer it to the next iteration of the Process.
